pkg/controllers/vdb/podfacts.go

```diff
@@ -566,8 +570,14 @@ func (p *PodFacts) checkForSimpleGatherStateMapping(_ context.Context, _ *vapi.V
 	pf.localDataSize = gs.LocalDataSize
 	pf.localDataAvail = gs.LocalDataAvail
 	pf.admintoolsExists = gs.AdmintoolsExists
-	// If the vertica process is running, then the database is UP. This is
-	// consistent with the liveness probe, which goes a bit further and checks
+	if vmeta.UseVClusterOps(vdb.Annotations) {
+		// In vclusterops mode, we call an HTTPS endpoint.
+		// If that returns HTTP code 200, then vertica is up
+		pf.upNode = gs.UpNode
+		return nil
+	}
+	// For admintools, if the vertica process is running, then the database is UP.
+	// This is consistent with the liveness probe, which goes a bit further and checks
 	// if the client port is opened. If the vertica process dies, the liveness
 	// probe will kill the pod and we will be able to do proper restart logic.
 	// At one point, we ran a query against the nodes table. But it became
```

Comment:
Can we go through this file and look at the usage of VerticaPIDRunning? For cases where we use it as a proxy for the db being up, we should just check upNode state.